### ISSUE TITLE
Commands in teardown_file won't be executed

### ISSUE THREAD

**Issue by badamowicz** [bats-core/bats-core#916]
This might not be a bug but simply a missunderstanding on my side. Some of my tests leave temporary files behind which I want to delete with the `teardown_file` function. However, none of the files will be deleted.

**Steps to reproduce the behavior**:

1. Create example `file.bats` with following contents
```bash
@test "create a file" {

  touch sample.file
}

teardown_file()  {

  rm sample.file
}
```
2. run `bats file.bats`

**Expected behavior**
The file `sample.file` is deleted.

**Environment:**
 - Bats version
```bash
$ bats -v
Bats 1.2.0-dev
```
 - operating system (including version):
```bash
$ uname -a
Linux bernd-vm 5.15.0-78-generic #85~20.04.1-Ubuntu SMP Mon Jul 17 09:42:39 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```
 - `bash --version`:
```bash
$ bash --version
GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
```
 - Install method: [e.g. git submodule, distribution package manager, npm, homebrew, ...]
     * installed as a submodule

Thanks in advance for helping!

**Comment by martin-schulze-vireso** [comment]
Version 1.2.0 is rather old. I believe it does not offer setup_file. Have you tried a newer version?

**Comment by badamowicz** [comment]
Yes, you're right. I accidentially used the bats binary that came with the OS (which is 1.2.0) and not the one from the submodule (which is 1.11.0). With 1.11.0 it works perfectly. Problem is solved. Thanks a lot!
